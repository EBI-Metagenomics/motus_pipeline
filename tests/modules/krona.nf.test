nextflow_process {

    name "Test module krona"
    script "modules/krona.nf"
    process "KRONA"

    test("krona") {

        when {
            params {
                otu = "${baseDir}/tests/modules/fixtures/mapseq2biom/output/test-mapseq.txt"
            }
            process {
                """
                input[0] = channel.fromPath("${params.otu}", checkIfExists: true)
                """
            }
        }

        then {
            assert process.success
            assert snapshot(process.out).match()
        }

    }
}