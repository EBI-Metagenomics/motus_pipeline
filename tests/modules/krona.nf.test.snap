{
    "krona": {
        "content": [
            {
                "krona_html": [
                    "krona.html:md5,32b3e1c37308d0fbc96993d527fe9f3c"
                ]
            }
        ],
        "timestamp": "2023-02-28T22:43:06+0000"
    }
}